### ISSUE TITLE
The code does not correspond to the paper

### ISSUE THREAD

**Issue by dengzuxuan** [Frank-Wang-oss/TS-GAC#7]
In Graph-Aware Contrasting for Multivariate Time-Series Classification paper say: in MWTC loss function,the positive pair in denominator , and negative pairs in denominator 
![paper](https://github.com/user-attachments/assets/558081c0-4fe4-451f-a646-7577162c77bf)
but I can't find  MWTC loss negative pairs in code,Please tell me where to find the correct function information. Thank you

        nce = 0  # average over timestep and batch
        encode_samples = torch.empty((self.timestep, batch*num_nodes, self.num_channels)).float().to(self.device) 
        for i in np.arange(1, self.timestep + 1):
            encode_samples[i - 1] = z_aug2[:, t_samples + i, :].view(batch*num_nodes, self.num_channels)
        forward_seq = z_aug1[:, :t_samples + 1, :]
        # print('forward_seq', forward_seq.size())

        c_t = self.seq_transformer(forward_seq)
        # print('c_t', c_t.size())

        pred = torch.empty((self.timestep, batch*num_nodes, self.num_channels)).float().to(self.device)
        for i in np.arange(0, self.timestep):
            linear = self.Wk[i]
            pred[i] = linear(c_t)
        for i in np.arange(0, self.timestep):
            total = torch.mm(encode_samples[i], torch.transpose(pred[i], 0, 1))
            nce += torch.sum(torch.diag(self.lsoftmax(total)))

        nce /= -1. * batch * num_nodes * self.timestep

**Comment by 1955515583-spec** [comment]
Have you solved this problem?
